apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -49,3 +56,15 @@ const AnimationPickerBody = React.createClass({
   }
 });
 export default Radium(AnimationPickerBody);
+
+const WarningLabel = ({children}) => (
+    <span style={warningLabelStyle}>
+      {children}
+    </span>
+);
+WarningLabel.propTypes = {
+  children: React.PropTypes.node
+};
+const warningLabelStyle = {
+  color: color.red
+};
```

Comment:
I would personally declare the style before using it. For something as simple as this, you could potentially just do it entirely inline too (I think our styleguide says that's okay if the style is only one field)